docs/migrate/migrate-guide.rst

```diff
@@ -75,6 +75,48 @@ Why use Qiskit Runtime?
 .. figure:: ../images/table.png
    :alt: table comparing backend.run to Qiskit Runtime primitives
 
+Key:
+
+- :octicon:`x` Not supported
+- :octicon:`check` Full support
+- :octicon:`clock` Future support
+
+
+.. list-table::
+  :header-rows: 1
```

Comment:
Isn't this a duplicate of the image on line 75?

Reply:
Oops!  Yes - I had the image in there in case I didn't get the table to work, then forgot to take it out!